src/core/bittorrent/session.cpp

```diff
@@ -2141,6 +2141,15 @@ void Session::handleAddTorrentAlert(libtorrent::add_torrent_alert *p)
             }
         }
 
+        if (pref->isAddTrackersEnabled() && !torrent->isPrivate()) {
+            QList<BitTorrent::TrackerEntry> trackers;
+            QStringList trackersList = pref->getTrackersList().split(QString::fromUtf8("\n"));
+            foreach (const QString &tracker, trackersList)
```

Comment:
You should populate ```trackers``` in Session::configure() (create member variable m_additionalTrackers).